routing/index_graph.cpp

```diff
@@ -23,10 +23,12 @@ m2::PointD const & IndexGraph::GetPoint(Joint::Id jointId)
   return m_geometry.GetPoint(m_jointIndex.GetPoint(jointId));
 }
 
+void IndexGraph::Build(uint32_t numJoints) { m_jointIndex.Build(m_roadIndex, numJoints); }
+
 void IndexGraph::Import(vector<Joint> const & joints)
 {
   m_roadIndex.Import(joints);
-  m_jointIndex.Build(m_roadIndex, joints.size());
+  Build(joints.size());
```

Comment:
Please, static_cast joints.size() to uint32_t. Also, feel free to add a CHECK that joints.size() is less than uint32_t.